Dockerfile.template

```diff
@@ -6,12 +6,12 @@ ENV MARIADB_VERSION=%%VERSION_FULL%%
 
 USER root
 
-RUN	apt-get update && \
-	apt-key adv --recv-keys \
+RUN	apt-key adv --recv-keys \
 		--keyserver hkp://keyserver.ubuntu.com:80 \
 		0xF1656F24C74CD1D8 && \
 	add-apt-repository "deb http://archive.mariadb.org/mariadb-${MARIADB_VERSION}/repo/ubuntu/ $(lsb_release -sc) main" && \
-	apt-get install mariadb-server gosu
+	apt-get install mariadb-server gosu && \
+	rm -rf /var/lib/apt/lists/*
```

Comment:
It's a best practice for our images as well as Docker as a whole to remove the package index in non-interactive images. This can make the image smaller (only 0.03GB in this case) but more importantly, makes the layer more deterministic which helps with layer caching in the long run.